crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -78,6 +102,19 @@ impl ActionHandler for BridgeLockAction {
             .await
             .context("failed to execute bridge lock action as transfer action")?;
 
+        // the transfer fee is already deducted in `transfer_action.execute()`,
+        // so we just deduct the bridge lock byte multiplier fee.
+        let byte_cost_multiplier = state
+            .get_bridge_lock_byte_cost_multiplier()
+            .await
+            .context("failed to get byte cost multiplier")?;
+        let fee = byte_cost_multiplier * DEPOSIT_BYTE_LEN;
```

Comment:
```suggestion
        let fee = byte_cost_multiplier.saturating_mul(DEPOSIT_BYTE_LEN);
```
And unit test please.